docs/src/add-services/_index.md

```diff
@@ -121,6 +121,13 @@ Add them to the `type` key of the [service configuration](#1-configure-the-servi
 <!-- To update the versions in this table, use docs/data/registry.json -->
 {{< readFile file="src/registry/images/tables/services_supported.md" markdownify="true">}}
 
+### Service versions
+
+These services generally follow [semantic versioning conventions](https://semver.org/).
+You choose which major/minor version to add to your project.
+Patch versions are applied periodically for bug fixes and the like.
```

Comment:
:hammer_and_wrench:
Although "Periodically" is a better fit, maybe adding the differentiation between what the customer has power over (and what not), might be helpful?
Since patches are applied by us, and they shouldn't be a direct concern for the customers.

```suggestion
Patch versions are applied automatically over time for bug fixes and the like.
```

Reply:
I considered moving away from the original `periodically` wording, but the thing is that the updates aren't applied automatically. The process on our side is manual. I feel like using `automatically` gives the impression that as soon as a patch is released it'll automatically be added to our versions.

Reply:
Fair point too... hmm, should we add more clarification, or good enough for now?

Reply:
I personally think it's clear enough for now. But if you feel differently or have another suggestion, we can look into it more.